modules/mono/editor/script_templates/EditorScenePostImport/basic_import_script.cs

```diff
@@ -7,12 +7,12 @@
 [Tool]
 public partial class _CLASS_ : _BASE_
 {
-    public override Object _PostImport(Node scene)
+    public override Godot.Object _PostImport(Node scene)
     {
         // Modify the contents of the scene upon import. For example, setting up LODs:
-//      scene.GetNode<MeshInstance3D>("HighPolyMesh").DrawDistanceEnd = 5.0
-//      scene.GetNode<MeshInstance3D>("LowPolyMesh").DrawDistanceBegin = 5.0
-        return scene // Return the modified root node when you're done.
+        // scene.GetNode<MeshInstance3D>("HighPolyMesh").DrawDistanceEnd = 5.0;
+        // scene.GetNode<MeshInstance3D>("LowPolyMesh").DrawDistanceBegin = 5.0;
```

Comment:
Would uncommenting in an IDE with a hotkey remove both `//` and the space? I think this was written in a way to be easy uncomment-able as a block.

Reply:
Hm indeed I missed that. I think it's a bad example if it's not relying on actually existing API, it's probably best to remove it from both templates.

Reply:
Done.